client/helper.go

```diff
@@ -77,6 +78,11 @@ func getRequest(clientCtx client.Context, id uint64) (*oracletypes.QueryRequestR
 	return queryClient.Request(context.Background(), &oracletypes.QueryRequestRequest{RequestId: id})
 }
 
+func getSigningResult(clientCtx client.Context, signingID uint64) (*bandtsstypes.QuerySigningResponse, error) {
+	queryClient := bandtsstypes.NewQueryClient(clientCtx)
+	return queryClient.Signing(context.Background(), &bandtsstypes.QuerySigningRequest{SigningId: signingID})
+}
```

Comment:
function name is `getSigningResult` but it returns `QuerySigningResponse`, please change return type to either convert `SigningResponse` to `SigningResult` or change function name

Reply:
change to getSigning to be align with getRequest naa.